### DISCUSSION TITLE
Backend/Whatever Developer

### DISCUSSION THREAD

**Discussion by pooyakn** [WeAreMahsaAmini/Core#199] [Introductions]
I'm Pooya, and I'd be happy to help you create web services using Golang/Node.JS.

If we need to develop things other than web services, I would still be happy to help.

I also have some experience (2y) with React before. Although my knowledge is somewhat outdated (1.5 years back), I can still help.

**Comment by ghost** [comment]
https://github.com/WeAreMahsaAmini/Core/discussions/256

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163
